content/blog/2022-05-17-aws-remotes-in-dvc.md

```diff
@@ -0,0 +1,147 @@
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - DVC Remotes
+  - Collaboration
+---
+
+When you’re working on a data science project that has huge datasets, it’s
+common to store them in cloud storage. You’ll also be working with different
+versions of the same datasets to train a model, so it’s crucial to have a tool
+that enables you to do this quickly and easily. That’s why we’re going to do a
+quick walkthrough of how to set up a remote in an AWS S3 bucket and handle data
+versioning with [DVC](https://dvc.org/doc).
+
+We’ll start by creating a new S3 bucket in our AWS account, then we’ll show how
+you can add DVC to your project, and finally, we’ll make updates to the dataset
+with DVC commands. We’ll be working with
+[this repo](https://github.com/iterative/stale-model-example) if you want an
+example to play with. By the time you finish, you should be able to create this
+setup for any ML project using an AWS remote.
+
+## Set up an AWS S3 bucket
+
+Make sure that you already have an [AWS account](https://aws.amazon.com/) and
+log in. Search for `S3` and it should be the first service that appears.
+
+![S3 service in AWS](/uploads/images/2022-05-17/finding_s3.png)
+
+Once you’re on the S3 page, click the `Create Bucket` button and it will take
+you to a page that looks like this. The bucket in this example is called
+`updatedbikedata` because that is the data our demo repo works with. You can
+leave the default settings in place or you can update them to fit the
+functionality you need.
+
+![create an S3 bucket in AWS](/uploads/images/2022-05-17/create_bucket.png)
+
+Once you’ve created the bucket, you should be redirected to the S3 dashboard and
+see the success message and your new bucket.
+
+![newly created S3 bucket in AWS](/uploads/images/2022-05-17/created_bucket.png)
+
+### Get your credentials
+
+Now that the S3 bucket is ready, we need the `access_key_id` and
+`secret_access_key` from AWS in order to connect to our project. You can create
+these keys in your Identity and Access Management settings. Go to your security
+credentials and select the `Access keys` section. Then click the
+`Create New Access Key` button. This will generate a new set of keys for you so
+make sure you download this file to get your secret access key.
+
+![make AWS access credentials](/uploads/images/2022-05-17/make_credentials.png)
+
+Once you’ve downloaded the credentials, you should see the access key ID in the
+table. Note that you won’t be able to access your secret key again at this
+point. You would need to make a new set of credentials if you don’t have it.
+
+![successfully created AWS access credentials](/uploads/images/2022-05-17/credentials.png)
+
+That’s it for setting up your bucket and getting the credentials you need! Now
+let’s add DVC to our demo repo and set up the remote.
+
+## Set up a DVC project
+
+First, add DVC as a requirement to your project with the following installation
+command:
+
+`$ pip install 'dvc[s3]'`
```

Comment:
Should all these be fenced code blocks? Easier for copy/paste

````suggestion
```dvc
$ pip install 'dvc[s3]'
```
````